src/Microsoft.AspNet.Razor/CodeGenerators/CSharpCodeWriter.cs

```diff
@@ -470,23 +470,6 @@ private void WriteCStyleStringLiteral(string literal)
                         Write(literal[i].ToString());
                         break;
                 }
```

Comment:
Chose to not remove the length restrictions to not muck with all the generated files in this PR. Also because the same code paths will be touched in [this](https://github.com/aspnet/Razor/issues/635) issue.